util/envDynawo.sh

```diff
@@ -1304,8 +1304,10 @@ nrt() {
   if ! is_launcher_installed; then
     install_launcher || error_exit "Error during launcher installation."
   fi
+  pushd "$DYNAWO_HOME" > /dev/null
```

Comment:
Why did you need to update this file with this development?

Reply:
This is not related to the development but was useful for my validation: CombiTable using files as inputs use paths relative to DYNAWO_HOME, so from version-validation or if you launch the nrt from another path than DYNAWO_HOME some tests were failing. Those lines make sure that the current path is dynawo home when nrt are launched to avoid those false positives

Reply:
ok thanks for the explanation :)